ui/src/project/patch.ts

```diff
@@ -0,0 +1,80 @@
+import type { Project } from "../project";
+import * as source from "../source";
+import type { Urn } from "../urn";
+import * as proxy from "../proxy";
+
+export type { Patch } from "../proxy";
+
+export interface PatchDetails {
+  patch: proxy.Patch;
+  commits: source.GroupedCommitsHistory;
+}
+
+export const tagPrefix = "radicle-patch/";
```

Comment:
In which cases do we write constants camelCase and in which ALL_CAPS, like here 👇 ? https://github.com/radicle-dev/radicle-upstream/blob/a86273751c4a1299617c0be7a59e5a48df28e84c/ui/src/config.ts#L4

Reply:
Fixed in https://github.com/radicle-dev/radicle-upstream/pull/1682/commits/82b4a6e091ea888026ff690128beda1a22900d3c